python/ray/data/datasource/file_meta_provider.py

```diff
@@ -324,3 +327,25 @@ def prefetch_file_metadata(
             return _fetch_metadata_remotely(pieces)
         else:
             return _fetch_metadata(pieces)
+
+
+def _handle_read_s3_files_error(error: OSError, paths: Union[str, List[str]]) -> str:
+    # Specially handle AWS error when reading files, to give a clearer error message
+    # to avoid confusing users. The real issus is most likely to be AWS S3 file
+    # credential not haven been properly configured yet.
```

Comment:
```suggestion
    # to avoid confusing users. The real issue is most likely that the AWS S3 file
    # credentials have not been properly configured yet.
```

Reply:
sorry, typing too fast, updated.